### DISCUSSION TITLE
21.08.11.수 회의록

### DISCUSSION THREAD

**Discussion by damilog** [Team-IT-DA/Frontend#64] [회의록]
- [x] 상품 페이지 내 장바구니 버튼 추가 여부(데이지) -> 빼는 걸로
- [x]  이번주 업무 배분

### 데이지
- 페이지 네이션(18개)
- 장바구니 페이지 뷰(시간 되는대로)
- 상품 탭 눌렀을 때 상품페이지로 이동하도록 라우팅 
- 상세 페이지로 이동하도록 라우팅
### 디코
- 장바구니 삭제 버튼 기능 구현
- 홈페이지에 있는 브랜드 스토리 삭제, 브랜드 이야기 탭 클릭시 브랜드 페이지로 이동 (라우팅)
- 브랜드 페이지에 헤더달기
- 회원가입 뷰(시간 되는대로)
- 헤더 리팩토링
- top button 붙여보기 (시간되면)
`<Header color={"#555555"} />`
### 제니
- mock data 만들어서 포토후기, 일반후기 필터링 
- 가격에 쉼표 붙이기
- 유의사항? 추가하는 것
- top button 붙여보기 (시간되면)
- AddProductPage 뷰(시간 되는대로)

**Comment by wjddnjswjd12** [comment]
늘 회의록 작성해주셔서 감솸돠 여러분...😁든든하네요..ㅠㅠ

**Reply by damilog** [reply]
진짜 회의록 넘 좋아요!!

**Reply by ha3158987** [reply]
맞아용 확인하기도 편하고 넘 좋네요!!👍😃 
wiki에 이번 회의록도 추가해뒀습니다~!
